### DISCUSSION TITLE
importation - a profile by "type"

### DISCUSSION THREAD

**Discussion by Laurent-Cosmic** [FLEXIcontent/flexicontent-cck#1208] [Ideas]
Hello guys
As I have a lot of fields to complete for each article, I'm using a lot the import function.
Is it possible to have a kind of save "profile" by type for instance.
For instance, for some types, I need to have metadesc, for some other not; for some, I need to have the current date when for some other, there is a column for this one, category is of course also different for the different types (some are also overrides but not  all of them)
and this only example....  
Thanks in advance to take few minutes to think about this idea

**Comment by micker** [comment]
hello it can be a good idea but for now we need to finish news version no feature will be added

**Reply by Laurent-Cosmic** [reply]
of course :D almost that this one if more for make my life easier than technical
